ext/B/t/concise.t

```diff
@@ -71,9 +70,6 @@ is ($@, '', "walk_output() accepts obj that can print");
 
 # test that walk_output accepts a HANDLE arg
 SKIP: {
-    skip("no perlio in this build", 4)
-        unless $Config::Config{useperlio};
-
```

Comment:
If we're removing the `skip()` function, shouldn't we also remove the `SKIP: { }` block (and de-intent the remaining code as we're doing in `ext/B/t/o.t` farther down)?

Reply:
Yes, whoops, I should have removed the `SKIP:` label. (I did in other places.) Well spotted- thanks. I did this by amending the existing commit, as I thought it would clutter the history if I did it as another commit.

As to re-indent, I wouldn't argue that we should *automatically* re-indent, if there's a lot of version history that it's going to conceal - ie reading through that would be harder than reading a block.

But in this case, yes, it seems that nothing substantive changed in these tests since 2004, and prior to then there isn't a lto of bug-fix detail in different commits.

So yes, something I should have thought about and done first time. I've done it as a separate commit, so that it's clear what is going on, and re-ordered it immediately after the commit that removes the `SKIP:` labels.

I rebased the branch onto blead and re-pushed. I don't know if that confuses GitHub...